{"uid":"93b5a6a380429ade","name":"test_raw_properties_list_status_02_551","fullName":"tests.test_checklist.TestPromotionChecklist#test_raw_properties_list_status_02_551","historyId":"f58d1ebb1721ce0b8104ab6edb75e5bf","time":{"start":1739801588612,"stop":1739801599340,"duration":10728},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id=\"nav-home\"]/div/table/tbody/tr[2]/td/div/input\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55962e9f509a <unknown>\n#1 0x55962e4bc8b0 <unknown>\n#2 0x55962e50db99 <unknown>\n#3 0x55962e50dd21 <unknown>\n#4 0x55962e55c854 <unknown>\n#5 0x55962e5339ad <unknown>\n#6 0x55962e559b8b <unknown>\n#7 0x55962e533753 <unknown>\n#8 0x55962e4ff38e <unknown>\n#9 0x55962e500b51 <unknown>\n#10 0x55962e9be76b <unknown>\n#11 0x55962e9c26f2 <unknown>\n#12 0x55962e9aa8fc <unknown>\n#13 0x55962e9c32e4 <unknown>\n#14 0x55962e98e9cf <unknown>\n#15 0x55962e9e3cd8 <unknown>\n#16 0x55962e9e3eb6 <unknown>\n#17 0x55962e9f3f16 <unknown>\n#18 0x7f5c6509caa4 <unknown>\n#19 0x7f5c65129c3c <unknown>","statusTrace":"tests/test_runner.py:23: in setUp\n    self._login()\ntests/test_runner.py:41: in _login\n    LoginComponent(self.driver).click_authorisation_btn()\nSLP/ui/PageObjects/SLPlogin/slp_login.py:20: in click_authorisation_btn\n    self.get_authorisation_btn().click()\nSLP/ui/PageObjects/SLPlogin/slp_login.py:16: in get_authorisation_btn\n    return self.node.find_element(*LOGIN_WITH_GOOGLE)\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:770: in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:384: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fd516a40050>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"no such element\",\"message\":\"no such element: Unable to locate element: {\\...\\\\n#17 0x55962e9f3f16 \\\\u003Cunknown>\\\\n#18 0x7f5c6509caa4 \\\\u003Cunknown>\\\\n#19 0x7f5c65129c3c \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id=\"nav-home\"]/div/table/tbody/tr[2]/td/div/input\"}\nE         (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nE       Stacktrace:\nE       #0 0x55962e9f509a <unknown>\nE       #1 0x55962e4bc8b0 <unknown>\nE       #2 0x55962e50db99 <unknown>\nE       #3 0x55962e50dd21 <unknown>\nE       #4 0x55962e55c854 <unknown>\nE       #5 0x55962e5339ad <unknown>\nE       #6 0x55962e559b8b <unknown>\nE       #7 0x55962e533753 <unknown>\nE       #8 0x55962e4ff38e <unknown>\nE       #9 0x55962e500b51 <unknown>\nE       #10 0x55962e9be76b <unknown>\nE       #11 0x55962e9c26f2 <unknown>\nE       #12 0x55962e9aa8fc <unknown>\nE       #13 0x55962e9c32e4 <unknown>\nE       #14 0x55962e98e9cf <unknown>\nE       #15 0x55962e9e3cd8 <unknown>\nE       #16 0x55962e9e3eb6 <unknown>\nE       #17 0x55962e9f3f16 <unknown>\nE       #18 0x7f5c6509caa4 <unknown>\nE       #19 0x7f5c65129c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py:232: NoSuchElementException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_TestPromotionChecklist","time":{"start":1739788254308,"stop":1739788254308,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id=\"nav-home\"]/div/table/tbody/tr[2]/td/div/input\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55962e9f509a <unknown>\n#1 0x55962e4bc8b0 <unknown>\n#2 0x55962e50db99 <unknown>\n#3 0x55962e50dd21 <unknown>\n#4 0x55962e55c854 <unknown>\n#5 0x55962e5339ad <unknown>\n#6 0x55962e559b8b <unknown>\n#7 0x55962e533753 <unknown>\n#8 0x55962e4ff38e <unknown>\n#9 0x55962e500b51 <unknown>\n#10 0x55962e9be76b <unknown>\n#11 0x55962e9c26f2 <unknown>\n#12 0x55962e9aa8fc <unknown>\n#13 0x55962e9c32e4 <unknown>\n#14 0x55962e98e9cf <unknown>\n#15 0x55962e9e3cd8 <unknown>\n#16 0x55962e9e3eb6 <unknown>\n#17 0x55962e9f3f16 <unknown>\n#18 0x7f5c6509caa4 <unknown>\n#19 0x7f5c65129c3c <unknown>","statusTrace":"tests/test_runner.py:23: in setUp\n    self._login()\ntests/test_runner.py:41: in _login\n    LoginComponent(self.driver).click_authorisation_btn()\nSLP/ui/PageObjects/SLPlogin/slp_login.py:20: in click_authorisation_btn\n    self.get_authorisation_btn().click()\nSLP/ui/PageObjects/SLPlogin/slp_login.py:16: in get_authorisation_btn\n    return self.node.find_element(*LOGIN_WITH_GOOGLE)\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:770: in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:384: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fd516a40050>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"no such element\",\"message\":\"no such element: Unable to locate element: {\\...\\\\n#17 0x55962e9f3f16 \\\\u003Cunknown>\\\\n#18 0x7f5c6509caa4 \\\\u003Cunknown>\\\\n#19 0x7f5c65129c3c \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id=\"nav-home\"]/div/table/tbody/tr[2]/td/div/input\"}\nE         (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nE       Stacktrace:\nE       #0 0x55962e9f509a <unknown>\nE       #1 0x55962e4bc8b0 <unknown>\nE       #2 0x55962e50db99 <unknown>\nE       #3 0x55962e50dd21 <unknown>\nE       #4 0x55962e55c854 <unknown>\nE       #5 0x55962e5339ad <unknown>\nE       #6 0x55962e559b8b <unknown>\nE       #7 0x55962e533753 <unknown>\nE       #8 0x55962e4ff38e <unknown>\nE       #9 0x55962e500b51 <unknown>\nE       #10 0x55962e9be76b <unknown>\nE       #11 0x55962e9c26f2 <unknown>\nE       #12 0x55962e9aa8fc <unknown>\nE       #13 0x55962e9c32e4 <unknown>\nE       #14 0x55962e98e9cf <unknown>\nE       #15 0x55962e9e3cd8 <unknown>\nE       #16 0x55962e9e3eb6 <unknown>\nE       #17 0x55962e9f3f16 <unknown>\nE       #18 0x7f5c6509caa4 <unknown>\nE       #19 0x7f5c65129c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py:232: NoSuchElementException","steps":[{"name":"Click authorisation button","time":{"start":1739801589279,"stop":1739801599339,"duration":10060},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id=\"nav-home\"]/div/table/tbody/tr[2]/td/div/input\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55962e9f509a <unknown>\n#1 0x55962e4bc8b0 <unknown>\n#2 0x55962e50db99 <unknown>\n#3 0x55962e50dd21 <unknown>\n#4 0x55962e55c854 <unknown>\n#5 0x55962e5339ad <unknown>\n#6 0x55962e559b8b <unknown>\n#7 0x55962e533753 <unknown>\n#8 0x55962e4ff38e <unknown>\n#9 0x55962e500b51 <unknown>\n#10 0x55962e9be76b <unknown>\n#11 0x55962e9c26f2 <unknown>\n#12 0x55962e9aa8fc <unknown>\n#13 0x55962e9c32e4 <unknown>\n#14 0x55962e98e9cf <unknown>\n#15 0x55962e9e3cd8 <unknown>\n#16 0x55962e9e3eb6 <unknown>\n#17 0x55962e9f3f16 <unknown>\n#18 0x7f5c6509caa4 <unknown>\n#19 0x7f5c65129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPlogin/slp_login.py\", line 20, in click_authorisation_btn\n    self.get_authorisation_btn().click()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPlogin/slp_login.py\", line 16, in get_authorisation_btn\n    return self.node.find_element(*LOGIN_WITH_GOOGLE)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[{"name":"Get authorisation button","time":{"start":1739801589280,"stop":1739801599338,"duration":10058},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id=\"nav-home\"]/div/table/tbody/tr[2]/td/div/input\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55962e9f509a <unknown>\n#1 0x55962e4bc8b0 <unknown>\n#2 0x55962e50db99 <unknown>\n#3 0x55962e50dd21 <unknown>\n#4 0x55962e55c854 <unknown>\n#5 0x55962e5339ad <unknown>\n#6 0x55962e559b8b <unknown>\n#7 0x55962e533753 <unknown>\n#8 0x55962e4ff38e <unknown>\n#9 0x55962e500b51 <unknown>\n#10 0x55962e9be76b <unknown>\n#11 0x55962e9c26f2 <unknown>\n#12 0x55962e9aa8fc <unknown>\n#13 0x55962e9c32e4 <unknown>\n#14 0x55962e98e9cf <unknown>\n#15 0x55962e9e3cd8 <unknown>\n#16 0x55962e9e3eb6 <unknown>\n#17 0x55962e9f3f16 <unknown>\n#18 0x7f5c6509caa4 <unknown>\n#19 0x7f5c65129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPlogin/slp_login.py\", line 16, in get_authorisation_btn\n    return self.node.find_element(*LOGIN_WITH_GOOGLE)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":2,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"_unittest_setUpClass_fixture_TestPromotionChecklist::0","time":{"start":1739801612961,"stop":1739801612961,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"@pytest.mark.filterwarnings('ignore::DeprecationWarning')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_checklist"},{"name":"subSuite","value":"TestPromotionChecklist"},{"name":"host","value":"fv-az1043-579"},{"name":"thread","value":"3351-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_checklist"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["@pytest.mark.filterwarnings('ignore::DeprecationWarning')"]},"source":"93b5a6a380429ade.json","parameterValues":[]}